proto/interchain_security/ccv/provider/v1/query.proto

```diff
@@ -23,3 +47,19 @@ message QueryConsumerGenesisResponse {
   interchain_security.ccv.consumer.v1.GenesisState genesis_state = 1
       [ (gogoproto.nullable) = false ];
 }
+
+message QueryConsumerChainsRequest {}
+
+message QueryConsumerChainsResponse { repeated string chain_ids = 1; }
```

Comment:
Accurate no :D

I'd argue it is still correct, though. In discussions with Simon we concluded that knowing the clientID might be useful and I reckon that renaming it would make it confusing.

`QueryConsumerChainsWithClientIdsResponse`  is a accurate and descriptive, but it may not add much clarity.

Reply:
@MSalopek well, in any case, the name of this field is wrong, since it is not a list of chain_ids.

Also, the handmade string format is not how we want to do things. It is weird, and will also cause problems if someone wants to parse what's coming out of this.

Make a `Chain` proto struct, that has a `chain id` and a `client id` field.

Reply:
`Done.`